src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__field {
+    width: $calendar-width;
+    display: flex;
+    gap: 1px;
+    flex-wrap: wrap;
+    padding: 100px 10px;
+    margin: 0 auto;
+
+    @for $length from 28 through $number-of-days {
```

put all 2 numbers into variables or leave them as just numbers

```suggestion
    @for $length from 28 through 31 {
```